shared/constants/chat2/convostate.tsx

```diff
@@ -79,13 +107,12 @@ export function getConvoState(id: Types.ConversationIDKey) {
 
 const Context = React.createContext<MadeStore | null>(null)
 
-type ConvoProviderProps = React.PropsWithChildren<{id: Types.ConversationIDKey}>
-export function Provider({children, ...props}: ConvoProviderProps) {
-  const storeRef = React.useRef<MadeStore>()
-  if (!storeRef.current) {
```

Comment:
we can't use this ref as changing the id needs to change the store. since `create` uses a map as a cache this will be idempotent so we can just do it directly